libpod/boltdb_state_internal.go

```diff
@@ -380,15 +380,17 @@ func (s *BoltState) getContainerFromDB(id []byte, ctr *Container, ctrsBkt *bolt.
 
 		ociRuntime, ok := s.runtime.ociRuntimes[runtimeName]
 		if !ok {
-			return errors.Wrapf(define.ErrInternal, "container %s was created with OCI runtime %s, but that runtime is not available in the current configuration", ctr.ID(), ctr.config.OCIRuntime)
+			err = errors.Wrapf(define.ErrRuntimeUnavailable, "cannot find OCI runtime %q for container %s", ctr.config.OCIRuntime, ctr.ID())
+			// fall back to the default runtime to allow ctr clean up
+			ociRuntime = s.runtime.defaultOCIRuntime
```

Comment:
This doesn't seem sane; we can't reasonably expect the default runtime to be able to do anything.